### PR TITLE
Files needed for openSUSE Chat

### PR DESCRIPTION
**Pull request by hellcp**

### CODE REVIEW

File: chat/welcome.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+}
+
+</style>
+
+<div class="mx_Parent">
+    <a href="https://chat.opensuse.org" target="_blank" rel="noopener">
+        <img src="//static.opensuse.org/chat/openSUSE-Chat-black-text.svg" alt="openSUSE Chat" class="mx_Logo"/>
+    </a>
+    <h1 class="mx_Header_title">Welcome to openSUSE Chat</h1>
+    <h4 class="mx_Header_subtitle">Chat for the user and contributors to the openSUSE Project</h4>
```
Comment (cboltz) [inline chat/welcome.html:168]: Minor nitpicking: We have more than one user ;-) therefore please make it `users`

**Comment by cboltz** [comment]
The SVG logos in `chat/integrations/` are all transparent, only the background/area around the logos is colored. This means that the SVG logos will have "random" colors depending on the background color of the page where they are used. I slightly doubt that this is intentional ;-)

For completeness: In the PNG versions, the logos are white instead of transparent.

**Comment by cboltz** [comment]
Sorry for not noticing earlier, but - most of the SVGs and PNGs in `chat/icons/` are also transparent.

**Comment by hellcp** [comment]
There it's intentional

**Comment by cboltz** [comment]
OK, then the last blocker is gone, and I'll merge ;-)

**Pull request merged**